app/src/test/java/org/mozilla/fenix/components/toolbar/DefaultBrowserToolbarControllerTest.kt

```diff
@@ -220,25 +220,19 @@ class DefaultBrowserToolbarControllerTest {
         val controller = createController()
         controller.handleToolbarClick()
 
-        val expected = BrowserFragmentDirections.actionGlobalSearchDialog(
+        val homeDirections = BrowserFragmentDirections.actionGlobalHome()
+        val searchDialogDirections = BrowserFragmentDirections.actionGlobalSearchDialog(
             sessionId = "1"
         )
 
-        verify { metrics.track(Event.SearchBarTapped(Event.SearchBarTapped.Source.BROWSER)) }
-        verify { navController.navigate(expected, any<NavOptions>()) }
-    }
-
-    @Test
-    fun handleToolbarClick_useNewSearchExperience() {
```

Comment:
We switched over to the "new" search experience a while ago, so this test is a dupe